src/util/native_fs_utils.js

```diff
@@ -336,8 +336,15 @@ async function delete_config_file(fs_context, schema_dir, config_path) {
         dbg.log1('native_fs_utils: delete_config_file unlinking:', config_path, 'is_gpfs=', is_gpfs);
         const tmp_dir_path = path.join(schema_dir, get_config_files_tmpdir());
         // TODO: add retry? should we fail deletion if the config file was updated at the same time?
-        await safe_unlink(fs_context, config_path, stat, gpfs_options, tmp_dir_path);
-
+        try {
+            await safe_unlink(fs_context, config_path, stat, gpfs_options, tmp_dir_path);
+        } catch (err) {
+            if (err.code === 'ENOENT') {
+                dbg.warn(`delete_config_file: config file already deleted ${config_path}`);
+                return;
```

Comment:
Why doesn't it include the `silent_if_missing` condition? (like we have in `folder_delete`).

Reply:
as we discussed, we always want on this case to not fail if the config file is missing